dockerfiles/Dockerfile.react-native

```diff
@@ -1,5 +1,5 @@
 # CI test image for unit/lint/type tests
-FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner:typed-platform-checks-ci
```

Comment:
Slightly concerned this made it onto master in the first place.  No real harm done, but would be good to get this merged asap.